src/index.js

```diff
@@ -42,10 +42,15 @@ export default function (h) {
       var isDeclsFunction = typeof decls === "function"
       !isDeclsFunction && (parsed = parse(decls))
       return function (data, children) {
-        data = data || {}
-        isDeclsFunction && (parsed = parse(decls(data)))
-        data.class = ((data.class || '') + ' ' + parsed).trim()
-        return h(tag, data, children)
+        isDeclsFunction && (parsed = parse(decls(data = data || {})))
```

Comment:
`data` can still be undefined here if `isDeclsFunction` is falsy

Reply:
Whoops.

Reply:
Thanks @maxholman! Fixed 👍